tests/unit/select_test.coffee

```diff
@@ -344,3 +344,22 @@ test 'optionValuePath with nested valuePath', ->
 
   equal obj2, select.get('selection'),
     'The right selection is retrieved'
+
+test 'shouldEnsureVisible controls whether to ensure visibility', ->
+  expect 2
+
+  select = @subject
+    content: ['foo']
+    ensureVisible: ->
+      ok yes, 'ensureVisible is called if shouldEnsureVisible is true'
+  select.set 'highlighted', 'foo'
+
+  select = @subject
+    content: ['bar']
+    shouldEnsureVisible: no
+
+  spy = sinon.spy select, 'ensureVisible'
+  select.set 'highlighted', 'bar'
+  equal spy.callCount, 0,
+    'ensureVisible is not called if shouldEnsureVisible is false'
+  spy.restore()
```

Comment:
not sure this is an improvement. First of all, it's not clear that each test only needs at most one spy; second of all this is a 350+ line file, while the spy is only for a 10 line test, which doesn't seem to worth a global variable.